tests/gis_tests/geoapp/tests.py

```diff
@@ -300,6 +300,13 @@ def test_crosses_lookup(self):
             0
         )
 
+    @skipUnlessDBFeature("supports_isvalid_lookup")
+    def test_is_valid_lookup(self):
```

Comment:
Here I mimicked the `isnull` lookup, which also uses the `is` language, I am not aware of any higher level rationale.